Client/Frontend/Browser/Playlist/PlaylistPopoverView.swift

```diff
@@ -0,0 +1,158 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+import Shared
+import BraveShared
+
+enum PlaylistPopoverState {
+    case addToPlaylist
+    case addedToPlaylist
+}
+
+struct PlaylistPopoverView: View {
+    @State
+    var state: PlaylistPopoverState
```

Comment:
I think we usually place `@State` keyword on the same line as property it applies to